app/javascript/src/components/editor/TranslationKeys/TranslationKeysEditStrings.svelte

```diff
@@ -47,19 +67,26 @@
 
 <div class="sticky top-0 well well--dark block p-1/4 mb-1/4">
   <div class="flex gap-1/4 align-center">
-    <input class="form-input" value={selectedKey} bind:this={renameInput} on:input={renameKey} />
+    <input
+      class="form-input"
+      value={createNewKeyMode ? "" : selectedKey}
+      bind:this={createOrRenameInput}
+      on:input={!createNewKeyMode && debounce(createOrRenameKey, 300)}
+      on:change={createNewKeyMode && createOrRenameKey}/>
     <button class="button button--danger button--small button--square" on:click={removeKey}>Remove</button>
   </div>
 
-  {#if error}
+  {#if createNewKeyMode}
+    <div class="text-orange mt-1/8">Give your translation a key</div>
```

Comment:
Perhaps a `transition:slide={{ duration: 100 }}` would look nice here, less jumpy

Reply:
Like so? Or just to the "Give your translation a key"?

![new key instruction and error slide animation](https://github.com/user-attachments/assets/07c009e9-838f-47ba-97ba-767a38910f14)

Reply:
Yeah that's nice!